CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Notable Changes
+
+- The async_operation_image property of the cumulus module should be updated to pull
```

Comment:
I think we need to move this into the v20 instructions - users need to make sure they're using v53 in that version and beyond.

Reply:
Ah, I see, you added it in 4006.

Reply:
This is in the unreleased section, but I also added a note in the v20 section.